### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by nevelate** [albi005/MaterialColorUtilities#34] [General]
I need to create material colors generator for Avalonia UI framework, can I use your projects source code?

**Comment by albi005** [comment]
Sure! But why not just use the nuget package?

**Reply by nevelate** [reply]
Thanks. I don't need maui related code.

**Reply by albi005** [reply]
The `MaterialColorUtilities` package doesn't have any maui specific code, those are in `MaterialColorUtilities.Maui`

**Reply by nevelate** [reply]
I got it.
